include/net/bsdlib.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @brief Initialize bsdlib.
+ *
+ * This function is synchrounous and it will block for
+ * --very long-- when the modem firmware is being updated.
```

Comment:
`very long` does not provide too much information, perhaps we could provide an approximate value here? A few minutes perhaps?

Reply:
Updated and extended.